README.md

```diff
@@ -80,66 +103,115 @@ Current adjacent projects that are within the TSC's scope of responsibility can
+The TSC exercises autonomy in setting up and maintaining procedures, policies,
+and management and administrative structures as it deems appropriate for the
+maintenance and operation of these projects and resources.
 
 Included in the responsibilities of the TSC are:
 
-* Managing code and documentation creation and changes for the listed projects and resources
-* Setting and maintaining standards covering contributions of code, documentation and other materials
-* Managing code and binary releases: types, schedules, frequency, delivery mechanisms
-* Making decisions regarding dependencies of the Node.js Core project, including what those dependencies are and how they are bundled with source code and releases
-* Creating new repositories and projects under the _nodejs_ GitHub organization as required
-* Setting overall technical direction for the Node.js Core project, including high-level goals and low-level specifics regarding features and functionality
-* Setting and maintaining appropriate standards for community discourse via the various mediums under TSC control
-* Setting and maintaining governance rules for the conduct and make-up of the TSC, CTC, Working Groups and other bodies within the TSC's domain
-
-Many of these responsibilities will be delegated by the TSC to appropriate bodies such as the CTC and Working Groups.
+* Managing code and documentation creation and changes for the listed projects
+  and resources
+* Setting and maintaining standards covering contributions of code,
+  documentation and other materials
+* Managing code and binary releases: types, schedules, frequency, delivery
+  mechanisms
+* Making decisions regarding dependencies of the Node.js Core project,
+  including what those dependencies are and how they are bundled with source
+  code and releases
+* Creating new repositories and projects under the _nodejs_ GitHub organization
+  as required
+* Setting overall technical direction for the Node.js Core project, including
+  high-level goals and low-level specifics regarding features and functionality
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+
+Many of these responsibilities will be delegated by the TSC to appropriate
+bodies such as the Working Groups.
 
 ## TSC Members
 
-TSC members are responsible for top level technical community concerns. The role is
-mostly administrative and is responsible for admitting new Top Level Projects, Top Level
-Working Groups, and advocating for any needs in the technical side of the foundation to
-the Node.js Foundation Board of Directors.
-
-TSC members can nominate new members at any time. Candidates for membership tend to be people
```

Comment:
Why has this paragraph been removed?

Reply:
Because it unnecessarily duplicates details from the TSC Charter.

Reply:
I think it is still valuable to have it stated here for posterity.

Reply:
How come? It's stated with greater clarity and detail in the charter and really does not serve much of a purpose.

Reply:
Unless it is stated somewhere else in this repo:
- it will be difficult to link to
- people will likely be unable to find how they could possibly become a TSC member in the broadest terms

Reply:
Very well.